### ISSUE TITLE
[exp] (_xdl || _load_xdl(...)).Project.getLatestReleaseAsync is not a function

### ISSUE THREAD

**Issue by julesmoretti** [expo/expo#1290]
ran the latest update using: `npm install -g exp` and now getting the following error when running `exp build:ios -c` or `exp build:ios`

```
[jmoretti:...somewhere/project]$ exp build:ios -c                                                                                                                                                                          (master✱) 
[exp] Checking if current build exists...

[exp] No currently active or previous builds for this project.
[exp] Removed existing credentials
? How would you like to upload your credentials?
 Expo handles all credentials, you can still provide overrides

We need your Apple ID/password to manage certificates and
provisioning profiles from your Apple Developer account.

Note: Expo does not keep your Apple ID or your Apple password.

? What's your Apple ID? myemailhere@somewhere.com
? Password? ***********
[exp] Validating Credentials...
[exp] You have 1 teams
Only 1 team associated with your account, using Team ID: AB123CD456
? Will you provide your own Distribution Certificate? Let Expo handle the process
? Will you provide your own Push Certificate? Let Expo handle the process
⠙ Now producing files for distCert
[exp] Looking for releases...
[exp] (_xdl || _load_xdl(...)).Project.getLatestReleaseAsync is not a function
[exp] Set EXPO_DEBUG=true in your env to view the stack trace.
```

Tried with new terminal windows etc...

expo version: 48.0.1
node: v7.0.0
npm: 4.6.1

I am not sure where EXPO_DEBUG is supposed to be set.

Thank you for your time and consideration

**Comment by jesseruder** [comment]
Thank you for reporting this! This should be fixed in `exp@48.0.2`.

**Comment by yoshi1125hisa** [comment]
I also experienced the same phenomenon.

```
~/Desktop $ exp init first-react-native-expo                                                                                                                                     1m  月 12/17 07:55:42 2018
We've built a brand new CLI for Expo!
Expo CLI is a drop in replacement for exp.
Install: npm install -g expo-cli
Use: expo --help
Read more: https://blog.expo.io/expo-cli-2-0-released-a7a9c250e99c
? Choose a template: blank
[07:55:59] (_xdl || _load_xdl(...)).Exp.downloadTemplateApp is not a function
[07:55:59] Set EXPO_DEBUG=true in your env to view the stack trace.
```
